micrograd/visitor.py

```diff
@@ -17,7 +17,8 @@ def transform(self, value: Value) -> ir.Module:
             with InsertionPoint(module.body):
                 @func.func()
                 def main():
-                    return arith.fptosi(ir.IntegerType.get_signless(32), self.walk(value))
+                    return self.walk(value)
```

Comment:
informational: originally i needed fptosi because i was returning it as the exit code.